core/controllers/ingest.py

```diff
@@ -45,6 +65,9 @@ def ingest():
     clean_data(workbook)
     populate_db(workbook, INGEST_MAPPINGS)
 
+    submission_id = workbook["Submission_Ref"].iloc[0]["Submission ID"]
```

Comment:
does this now need fetching from DB (or fetching as param from when it was created)?

Reply:
It's inserted into the dataframe so can grab it from here. We could return it from populate_db instead?

Reply:
nah, no preference, that's fine